tests/package_test.py

```diff
@@ -10,7 +10,7 @@
 from drift_protocol.common import DriftPackage, StatusCode, DataPayload
 from google.protobuf.any_pb2 import Any
 
-from drift_client.drift_client import Package
+from drift_client import Package
```

Comment:
update to new name, also, should it not be imported from `drift-client.package`?

Reply:
no, it is a unit test, and we import classes as a user should import them.